internal/transform/transaction_test.go

```diff
@@ -134,9 +134,11 @@ func makeTransactionTestInput() (transaction []ingest.LedgerTransaction, history
 							Text: &hardCodedMemoText,
 						},
 						Fee: 90000,
-						TimeBounds: &xdr.TimeBounds{
-							MinTime: 0,
-							MaxTime: 1594272628,
+						Cond: xdr.Preconditions{
```

Comment:
minor suggestion, could define some `ExtraSigners` to trigger coverage over that formatting routine.

Reply:
good point, i'll incorporate another test that validates more of the Protocol 19 preconditions